core/serial_command.py

```diff
@@ -81,12 +81,18 @@ class SerialCommand(object):
     """Contains a serial command object.
     """
 
-    def __init__(self, port_name=None):
+    def __init__(self, port_name=None, port_timeout=None):
         """Initialise the serial command"""
         if not port_name:
-            self._port_name = "/dev/tty.usbserial-FTGA2OCZ"
+            self._port_name = "/dev/ttyUSB0"
         else:
             self.logger = tellie_logger.TellieLogger.get_instance()
+            self._port_name = port_name
+            
+        if not port_timeout:
```

Comment:
Ahh, this relates to my above comment. So you don't actually set the port_timeout to None, but 0. This may be confusing to someone who has experience with pyserial.